plugins/pulp_rpm/plugins/importers/yum/sync.py

```diff
@@ -374,8 +378,10 @@ def check_metadata(self, url):
             # remember the reason so it can be reported to the user if no treeinfo is found either.
             self.repomd_not_found_reason = e.message
             _logger.debug(_('No yum repo metadata found.'))
+            self.skip_repomd_steps = True
```

Comment:
I'm not sure if there's anything useful you could add here as an in-line comment, but it's would potentially be very helpful. This bool gets used in enough places, it's hard to follow the logic flow. If there's nothing useful to add here, that's fine, but if it's possible to explain a little bit why they're changing here and 3 lines below, that would be helpful.